orca-clouddriver/src/main/java/com/netflix/spinnaker/orca/clouddriver/tasks/monitoreddeploy/MonitoredDeployBaseTask.java

```diff
@@ -269,28 +266,24 @@ private MonitoredDeployStageData getStageContext(StageExecution stage) {
   }
 
   @VisibleForTesting
-  String getRetrofitLogMessage(Response response) {
-    if (response == null) {
+  String getErrorMessage(SpinnakerServerException spinnakerException) {
+    if (!(spinnakerException instanceof SpinnakerHttpException)) {
       return "<NO RESPONSE>";
     }
 
-    String body = "";
-    String status = "";
-    String headers = "";
+    SpinnakerHttpException httpException = (SpinnakerHttpException) spinnakerException;
 
     try {
-      status = String.format("%d (%s)", response.getStatus(), response.getReason());
-      body =
-          CharStreams.toString(
-              new InputStreamReader(response.getBody().in(), StandardCharsets.UTF_8));
-      headers =
-          response.getHeaders().stream().map(Header::toString).collect(Collectors.joining("\n"));
+      String body = "";
+      if (httpException.getResponseBody() != null) {
+        body = objectMapper.writeValueAsString(httpException.getResponseBody());
+      }
+      return String.format("headers: %s\nresponse body: %s", httpException.getHeaders(), body);
     } catch (Exception e) {
-      log.error(
-          "Failed to fully parse retrofit error while reading response from deployment monitor", e);
+      log.error("Failed to fully read http error response details", e);
```

Comment:
```suggestion
      log.error("Failed to serialize http error response details", e);
```

Reply:
The error log is now is now reword to : "Failed to serialize http error response details"